ZenPacks/zenoss/Import4/bin/imp4_check.sh

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/env bash
```

Comment:
How resilient is this script?  What happens if I start a run and I kill it, then I want to re-initialize?